src/content/effective-dart/style.md

```diff
@@ -196,54 +194,80 @@ as in the following cases:
 We initially used Java's `SCREAMING_CAPS` style for constants. We
 changed for a few reasons:
 
-*   `SCREAMING_CAPS` looks bad for many cases, particularly enum values for
-    things like CSS colors.
-*   Constants are often changed to final non-const variables, which would
-    necessitate a name change.
-*   The `values` property automatically defined on an enum type is const and
-    lowercase.
+* `SCREAMING_CAPS` looks bad for many cases,
+  particularly enum values for things like CSS colors.
+* Constants are often changed to final non-const variables,
+  which would necessitate a name change.
+* The `values` property defined on an enum type is const and lowercase.
+
 :::
 
 [protobufs.]: {{site.pub-pkg}}/protobuf
 
 
 ### DO capitalize acronyms and abbreviations longer than two letters like words
 
-Capitalized acronyms can be hard to read, and
-multiple adjacent acronyms can lead to ambiguous names.
-For example, given a name that starts with `HTTPSFTP`, there's no way
-to tell if it's referring to HTTPS FTP or HTTP SFTP.
-
-To avoid this, acronyms and abbreviations are capitalized like regular words.
+Capitalized acronyms can be hard to read,
+and multiple adjacent acronyms can lead to ambiguous names.
+For example, given an identifier `HTTPSFTP`,
+the reader can't tell if it refers to `HTTPS` `FTP` or `HTTP` `SFTP`.
+To avoid this,
+capitalize most acronyms and abbreviations like regular words.
+This identifier would be `HttpsFtp` if referring to the former
+or `HttpSftp` for the latter.
 
-**Exception:** Two-letter *acronyms* like IO (input/output) are fully
-capitalized: `IO`. On the other hand, two-letter *abbreviations* like
-ID (identification) are still capitalized like regular words: `Id`.
+Two-letter abbreviations and acronyms are the exception.
+If both letters are capitalized in English,
+then they should both stay capitalized when used in an identifer.
+Otherwise, capitalize it like a word.
 
 ```dart tag=good
-class HttpConnection {}
-class DBIOPort {}
-class TVVcr {}
-class MrRogers {}
-
-var httpRequest = ...
-var uiHandler = ...
-var userId = ...
-Id id;
+// Longer than two letters, so always like a word:
+Http // "hypertext transfer protocol"
+Nasa // "national aeronautics and space administration"
+Uri // "uniform resource locator"
+Esq // "esquire"
+Ave // "avenue"
+
+// Two letters, capitalized in English, so capitalized in an identifier:
+ID // "identifier", capitalized as "ID"
+TV // "television"
+DB // "database", originally an acronym from "data base"
+IO // "input/output"
+UI // "user interface"
+
+// Two letters, not capitalized in English, so like a word in an identifier:
+Mr // "mister"
+St // "street"
+Rd // "road"
```

Comment:
Both this (and the "bad" list lines 246-260) seem really long. I'd suggest 2-3 examples per category max

Reply:
I can trim the middle list, but the first list has 4 and 3 letter examples that are helpful.